core-parser/src/test/java/org/uniprot/core/parser/tsv/uniparc/UniParcEntryLightValueMapperTest.java

```diff
@@ -0,0 +1,139 @@
+package org.uniprot.core.parser.tsv.uniparc;
+
+import org.junit.jupiter.api.Test;
+import org.uniprot.core.Location;
+import org.uniprot.core.Sequence;
+import org.uniprot.core.impl.SequenceBuilder;
+import org.uniprot.core.uniparc.*;
+import org.uniprot.core.uniparc.impl.*;
+import org.uniprot.core.uniprotkb.taxonomy.Organism;
+import org.uniprot.core.uniprotkb.taxonomy.impl.OrganismBuilder;
+
+import java.time.LocalDate;
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class UniParcEntryLightValueMapperTest {
```

Comment:
TSV is not supporting new fields created in field config. How do we want to handle it? Happy to chat about it